src/Microsoft.Health.Fhir.SqlServer/Features/Operations/Import/SqlImportOperation.cs

```diff
@@ -325,17 +364,56 @@ public async Task PostprocessAsync(CancellationToken cancellationToken)
             }
         }
 
-        private async Task<(string tableName, string indexName)> ExecuteRebuildIndexTaskAsync(string tableName, string indexName, bool pageCompression, CancellationToken cancellationToken)
+        private async Task<string> ExecuteSqlCommand(string tableName, string indexName, string command, CancellationToken cancellationToken)
```

Comment:
Nit: should the method name still reference reindex? This sounds like a generic method to run any sql command.

Reply:
Sorry, I will modify this function name.